src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -1062,6 +1062,7 @@ export default class UniversalDiscoveryModule extends Component {
                     onCancel={this.props.onCancel}
                     handlePublish={this.handlePublish}
                     restInfo={this.props.restInfo}
+                    setMainContainerRef={this.setMainContainerRef}
```

Comment:
I realised, after analysing _ContentCreatorComponent_, that we have here `setMainContainerRef={` and not `ref={`. It was confusing at first. Maybe cleaner would be to wrap _ContentCreatorComponent_ into forwardRef? But I'm not 100% sure. 🤔

Reply:
I didn't want to rewrite many files to follow the `forwardRef` convention. But, you're right in other cases I would go with `forwardRef` as it's a better approach. Here, I wanted to make it simple without to much rewriting.

Reply:
Later I also realised that, actually, we need two _refs_ to be set - we are also missing `setContentContainerRef`. So, we even cannot use forwardRef as it only works for one reference.